smoke_test/steps/base_step.rb

```diff
@@ -0,0 +1,41 @@
+module SmokeTest
+  module Steps
+    class BaseStep
+      include Capybara::DSL
+
+      attr_accessor :state
+
+      def initialize(state)
+        puts "Step:#{step_name}"
+        @state = state
+      end
+
+      def validate!
+        true
+      end
+
+      def complete_step
+        fail NotImplementedError
```

Comment:
I'll go ahead and implement.  We're ahead of the game and if we don't do it now...